### ISSUE TITLE
Maybe make taint a bit more dangerous?

### ISSUE THREAD

**Issue by Morinor** [Azanor/thaumcraft-suggestions#307]
Now that stabilizers allow you to prevent taint outbreaks even after a rift opened, maybe its time to make taint more dangerous again?
Im not saying it should be like in beta3 where it inevitably consumed your world, but right now it just feels a little bit too balanced, it spreads very slowly and is easy to get rid of.
(and its almost impossible to create a "stable" tainted area to farm vitium)
Maybe the it could spread exponentially faster? 
Currently you can have a taint seed near you and do nothing about it, at least for quite a while.
I feel like its just a little too balanced right now.
Also, maybe a player with high warp could speed up taint spread nearby, this could be a way to force more advanced players to keep an eye on their flux.
Basically, the more effective you can fight taint, the more dangerous it should be to be careless

**Comment by Geethebluesky** [comment]
Regarding spread, you can adjust that to be faster in the mod configuration.
I disagree that you can just ignore taint seeds though, unless Taint poison 3 was somehow nerfed recently? I haven't seen a rift in Beta 16 yet.

**Comment by Morinor** [comment]
Well, you have to get pretty close to get poisoned, and killing the seed with a bow is pretty easy.
I didnt say i want faster spread in general, i want it to be more dangerous to keep taint, and that players that are able to prevent flux events get punished more for being careless.

**Comment by Reassembly** [comment]
I dunno... people _still_ periodically talk about the severity of rifts/flux/taint - and the "fact" that there's "no way to fix it" - like it's a game-ruining bug. IMO, taint shouldn't be made _more_ dangerous until the (already-implemented) means of dealing with it are documented a little more explicitly in the Thaumonomicon. Currently, people seem to be failing to "solve the puzzle" of taint because they don't realize there _is_ a puzzle, or that it _has_ solutions.

Also, I had a "panic situation" a little while ago where my "taint patch" got big enough to have _two_ taint seeds, and and I couldn't seem to kill either of them permanently, at first - they just respawned a few blocks away after being killed. I thought it was a bug at first, but apparently, taint seeds can spawn _more_ taint seeds, _if_ there's tainted land that's far enough away from the first seed. I basically had to kill both seeds on opposite sides of the taint patch within a few seconds of each other to keep them from instantly reappearing - if there had been _three,_ I might have had to start building turrets and finding a way to remotely activate and/or synchronize them. So, there's already a _small_ element of "exponential danger", if the taint truly gets out of control.

On the other hand, I agree that high warp should have more effect on flux/taint. Currently, there's the "Thaumarhia" warp effect that drops puddles of flux goo, but that could stand to be much stronger. In addition, maybe the Flux Flu/Flux Phage effects, instead of merely increasing vis costs, could convert some or all of the "wasted" vis into flux? That might make using purely thaumaturgical methods to fight taint a bit more dangerous, if not counterproductive...
